### PR TITLE
Add AdoptOpenJDK 11 to CI

### PR DESCRIPTION
**Pull request by dwijnand**
... and cache all of ~/.sbt (particularly for ~/.sbt/launchers).

Refs #530.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,19 @@ scala:
...
 
+env:
+  matrix:
+    - TRAVIS_JDK=adopt@1.8.202-08
+    - TRAVIS_JDK=adopt@1.11.0-2
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: TRAVIS_JDK=adopt@1.11.0-2 # not fully supported but allows problem discovery
```
Comment (raboof) [inline .travis.yml]: Should we leave open #530 to fix these? Or introduce another ticket?

File: .travis.yml [hunk 2]
```diff
@@ -49,7 +62,8 @@ before_cache:
 cache:
   directories:
     - $HOME/.ivy2/cache
-    - $HOME/.sbt/boot
+    - $HOME/.jabba/jdk
+    - $HOME/.sbt
```
Comment (raboof) [inline .travis.yml]: :+1:

File: .travis.yml (hunk 1 above)
Comment (dwijnand) [reply .travis.yml]: I don't mind.  Which do you prefer?

Comment (raboof) [reply .travis.yml]: No preference - let's keep #530 open for this then?

**Pull request merged**